### PR TITLE
Jonchaney/styling

### PR DESCRIPTION
**Pull request by jonchaney**

### CODE REVIEW

File: frontend/app/components/banksend/banksend.js [hunk 1]
```diff
@@ -211,7 +222,7 @@ class BankSend extends Component {
         );
       }
       const usd = (
-        <Text style={styles.usd}>${Util.truncate(this.state.usd, 2)}</Text>
+        <Text style={styles.usd}>${this.state.usd}</Text>
```
Comment (pateldd1) [inline frontend/app/components/banksend/banksend.js]: Can’t reassign usd if it is const

File: frontend/app/components/wallet/personalWallet.js [hunk 2]
```diff
@@ -66,35 +67,32 @@ class PersonalWallet extends React.Component {
...
+                        <TouchableOpacity onPress={() => this.clipBoardCopy(this.props.personalSecret)}>
+                            {this.state.personalSecret ? <Text style={styles.addressFont}>secret key: {this.state.personalSecret}</Text> : null}
+                        </TouchableOpacity>
+                        {this.state.personalSecret ? <Text style={styles.noteFont}>Note: RipplePay does not store your Secret Key so please store it now.
+                                                      You will need it to make payments and it will disappear on page refresh.
+                        </Text> : null}
+                        
+                    <View style={styles.tab}>
+                        <SingleTab
+                            text="remove personal wallet"
```
Comment (pateldd1) [inline frontend/app/components/wallet/personalWallet.js]: Either you or me needs to add an alert that asks “are you sure?”

File: frontend/app/components/banksend/banksend.js [hunk 3]
```diff
@@ -144,19 +150,24 @@ class BankSend extends Component {
...
         </View>
       </View>
     );
   }
 
   passwordLock() {
+    let usd = Util.truncate(this.state.usd, 2);
+    console.log(this.state.usd);
+    if (this.state.usd === 0) {
+      usd = 0;
```
Comment (pateldd1) [inline frontend/app/components/banksend/banksend.js]: put this logic inside of Util.truncate. If 0 -> return 0

File: frontend/app/components/banksend/banksend.js (hunk 1 above)
Comment (pateldd1) [reply frontend/app/components/banksend/banksend.js]: No idea why this is still working. Just change to 'let'

File: frontend/app/components/banksend/banksend.js [hunk 4]
```diff
@@ -134,6 +135,11 @@ class BankSend extends Component {
   }
 
   topContainer() {
+    let balance = Util.truncate(this.props.balance, 2);
+    console.log(this.props.balance);
+    if (this.props.balance === 0) {
+      balance = 0;
```
Comment (pateldd1) [inline frontend/app/components/banksend/banksend.js]: Put this logic in Util.truncate

File: frontend/app/components/presentationals/singleTab.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  const topTab = {
+    fontSize: 13,
+    fontWeight: "600",
+    textAlign: 'center',
+    color: "red"
+  };
+  return (
+    <View style={styles.topTabsContainer}>
+      <TouchableOpacity style={topTabContainer}>
+        <Text onPress={props.handleLeftPress} style={topTab}>
```
Comment (pateldd1) [inline frontend/app/components/presentationals/singleTab.js]: rename to 'handlepress'

File: frontend/app/components/presentationals/singleTab.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
```
Comment (pateldd1) [inline frontend/app/components/presentationals/singleTab.js]: raname from topTabContainer to SingleTabContainer

File: frontend/app/components/banksend/banksend.js (hunk 1 above)
Comment (jonchaney) [reply frontend/app/components/banksend/banksend.js]: weird. I changed it to let, not sure why it was working either!

File: frontend/app/components/wallet/personalWallet.js (hunk 2 above)
Comment (jonchaney) [reply frontend/app/components/wallet/personalWallet.js]: i'll add it the in the next PR.

File: frontend/app/components/banksend/banksend.js (hunk 3 above)
Comment (jonchaney) [reply frontend/app/components/banksend/banksend.js]: changed it to a ternary operator

File: frontend/app/components/banksend/banksend.js (hunk 4 above)
Comment (jonchaney) [reply frontend/app/components/banksend/banksend.js]: changed it to a ternary operator

File: frontend/app/components/presentationals/singleTab.js (hunk 5 above)
Comment (jonchaney) [reply frontend/app/components/presentationals/singleTab.js]: changed

File: frontend/app/components/presentationals/singleTab.js (hunk 6 above)
Comment (jonchaney) [reply frontend/app/components/presentationals/singleTab.js]: changed

**Pull request merged**